### ISSUE TITLE
pypi doesn't contain `video` and other (nested) modules

### ISSUE THREAD

**Issue by stephematician** [tiagocoutinho/linuxpy#1]
As per title, perhaps these features are only intended for the development version?

```
>>> from linuxpy.video.device import Device
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'linuxpy.video'
```

**Comment by stephematician** [comment]
This line in `pyproject.toml` will disable automatic discovery (<https://setuptools.pypa.io/en/latest/userguide/package_discovery.html>):

```toml
[tool.setuptools]
packages = ["linuxpy"]
```

**Comment by tiagocoutinho** [comment]
Thanks for reporting. I could immediately see the error.
It has been fixed since release 0.4.0
